### PR TITLE
Dpp 384 todo cleanup postgres storage backend

### PR DESCRIPTION
**Pull request by nmarton-da**
### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Comment by nmarton-da** [comment]
Note for reviewers: best approach might be go commit by commit, separate commits are compiling and holding self-contained part of the work.
In the separate commit descriptions I tried to leave sufficient justifications if TODOs were just removed.

File: ledger/participant-integration-api/src/main/resources/db/migration/postgres/V51__drop_configuration_from_parameters_table.sha256 [hunk 1]
```diff
@@ -0,0 +1 @@
+6326123dd12a9d3652369df4ff68e7479716307b95960211aefa08fc8a6ce057
```
Comment (rautenrieth-da) [inline ledger/participant-integration-api/src/main/resources/db/migration/postgres/V51__drop_configuration_from_parameters_table.sha256:1]: Note to self: this file will have to be renamed to V101 or merged with the V100 migration after https://github.com/digital-asset/daml/pull/9481 is merged.

**Review by rautenrieth-da — APPROVED** [review]
Nice work removing the last configuration from the parameters table. Did you investigate whether it has any performance impact? Either a positive one because the heavily used parameters table is now smaller, or a negative one because the latest config lookup is now more expensive.

**Comment by nmarton-da** [comment]
> Nice work removing the last configuration from the parameters table. Did you investigate whether it has any performance impact? Either a positive one because the heavily used parameters table is now smaller, or a negative one because the latest config lookup is now more expensive.

Good point @rautenrieth-da , I did not. I do not expect any negative effect on the ingestion side (less data to insert). So what regression can happen is on the reads. I guess an LR run with some comparison with a baseline is in order, but there I might be a bit out of my depth. Let's discuss this on the next daily.

**Pull request merged**